server/constants/prisonInformation.ts

```diff
@@ -0,0 +1,30 @@
+type PrisonInformation = {
+  prisonName: string
+  prisonPhoneNumber: string
+  prisonWebsite: string
+}
+
+const prisonInformation: Record<string, PrisonInformation> = {
+  DHI: {
+    prisonName: 'Drake Hall (HMP)',
+    prisonPhoneNumber: '0121 661 2101',
+    prisonWebsite: 'https://www.gov.uk/guidance/drake-hall-prison',
+  },
+  FHI: {
+    prisonName: 'Foston Hall (HMP)',
+    prisonPhoneNumber: '0121 661 2101',
+    prisonWebsite: 'https://www.gov.uk/guidance/foston-hall-prison',
+  },
+} as const
```

Comment:
Do you need this when it's declared as a `const`?

Reply:
just following practices used in Staff UI (has same method, same way of definition etc)

Reply:
Probably me that originally put the `as const` in 😅 ! If I remember correctly, I *think* it stops the data being modified. E.g. if you have `const name = 'Tom'` then you can't change it. But if you had `const person = { name: 'Tom' }`, you can't change the whole object but you could do `person.name = 'xyz'` - which we wouldn't want.